web/themes/custom/fluffiness/templates/node--hospital.html.twig

```diff
@@ -0,0 +1,132 @@
+ *   - node--view-mode-[view_mode]: The View Mode of the node; for example, a
+ *     teaser would result in: "node--view-mode-teaser", and
+ *     full: "node--view-mode-full".
+ *   The following are controlled through the node publishing options.
+ *   - node--promoted: Appears on nodes promoted to the front page.
+ *   - node--sticky: Appears on nodes ordered above other non-sticky nodes in
+ *     teaser listings.
+ *   - node--unpublished: Appears on unpublished nodes visible only to site
+ *     admins.
+ * - title_attributes: Same as attributes, except applied to the main title
+ *   tag that appears in the template.
+ * - content_attributes: Same as attributes, except applied to the main
+ *   content tag that appears in the template.
+ * - author_attributes: Same as attributes, except applied to the author of
+ *   the node tag that appears in the template.
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ * - view_mode: View mode; for example, "teaser" or "full".
+ * - teaser: Flag for the teaser state. Will be true if view_mode is 'teaser'.
+ * - page: Flag for the full page state. Will be true if view_mode is 'full'.
+ * - readmore: Flag for more state. Will be true if the teaser content of the
+ *   node cannot hold the main body content.
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_node()
+ *
+ */
+#}
+
+{{ attach_library('fluffiness/hospitals') }}
+
+{% set classes = [
+  'node',
+  'node--type-' ~ node.bundle|clean_class,
+  node.isPromoted() ? 'node--promoted',
+  node.isSticky() ? 'node--sticky',
+  not node.isPublished() ? 'node--unpublished',
+  view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
+] %}
+
+<article{{ attributes.addClass(classes) }} style="background-color: {{ maria }}">
+
+  {% set classes2 = [
+    'title',
+    'node__title',
+  ] %}
+
+  {{ title_prefix }}
+  <h2 {{ attributes.addClass(classes2) }} {{ title_attributes }}>
+    <span>{{ label }}</span>
+  </h2>
+  {{ title_suffix }}
+
+  {#  content with sidebar #}
+
+  <div{{ content_attributes.addClass('node__content') }}>
+    {{ content|without(
+      'body',
+      'field_built_date',
+      'field_colour',
+      'field_header_image',
+      'field_numer_of_patients',
+      'field_phone_number',
+      'field_region'
+    ) }}
+    <div class="hospital__date">
+      {{ content.field_built_date }}
+    </div>
+    <hr>
+    <div class="hospital__image">
+      {{ content.field_header_image }}
+      <div class="hospital__image-alt">
+        {{ content.field_header_image['#items'].alt }}
```

Comment:
why is the image `alt` displayed separately?

Reply:
Because it was not displayed by default and this was the solution I came up with 🙂. In the manage fields I can only show the label, but not the alt text...